README.md

```diff
@@ -67,16 +68,68 @@ You can pass the following options to `cmake ..`.
+
+## Run
+### GStreamer Plugin (kvssink)
+
+#### Loading Element
+The GStreamer plugin will be in your `build` directory, to load this plugin it will need to be in your `GST_PLUGIN_PATH`. Do this you can run
+
+```
+export GST_PLUGIN_PATH=`pwd`/build
+```
+
+Now if you execute `gst-inspect-1.0 kvssink` you should get information on the plugin like
+
+```text
+Factory Details:
+  Rank                     primary + 10 (266)
+  Long-name                KVS Sink
+  Klass                    Sink/Video/Network
+  Description              GStreamer AWS KVS plugin
+  Author                   AWS KVS <kinesis-video-support@amazon.com>
+
+Plugin Details:
+  Name                     kvssink
+  Description              GStreamer AWS KVS plugin
+  Filename                 /Users/seaduboi/workspaces/amazon-kinesis-video-streams-producer-sdk-cpp/build/libgstkvssink.so
+  Version                  1.0
+  License                  Proprietary
+  Source module            kvssinkpackage
+  Binary package           GStreamer
+  Origin URL               http://gstreamer.net
+```
+
+If the build failed, or `GST_PLUGIN_PATH` is not proprely set you will get output like
+
+```text
+No such element or plugin 'kvssink'
+```
+
+
+#### Using Element
+The kvssink element has the following required parameters:
+
+* `stream-name` -- The name of the destination Kinesis video stream.
+* `storage-size` -- The storage size of the device in kilobytes. For information about configuring device storage, see StorageInfo.
+* `access-key` -- The AWS access key that is used to access Kinesis Video Streams. You must provide either this parameter or credential-path.
+* `secret-key` -- The AWS secret key that is used to access Kinesis Video Streams. You must provide either this parameter or credential-path.
+* `credential-path` -- A path to a file containing your credentials for accessing Kinesis Video Streams. For example credential files, see Sample Static Credential and Sample Rotating Credential. For more information on rotating credentials, see Managing Access Keys for IAM Users. You must provide either this parameter or access-key and secret-key.
 
-### GStreamer sample
-To generate GStreamer samples in the `build` directory, run the following from the build directory:
 
-`cmake .. -DBUILD_GSTREAMER_PLUGIN=ON`; `make`
+For examples of common use cases you can look at [Example: Kinesis Video Streams Producer SDK GStreamer Plugin](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/examples-gstreamer-plugin.html)
 
 ## Documentation
 
 ## Related
+* [What Is Amazon Kinesis Video Streams](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/what-is-kinesis-video.html)
+* [C SDK](https://github.com/awslabs/amazon-kinesis-video-streams-producer-c)
+* [Example: Kinesis Video Streams Producer SDK GStreamer Plugin](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/examples-gstreamer-plugin.html)
 
```

Comment:
Looks good to me. Just a couple of comments: 

1. Can you add this line in the README.md as well? 
"If cmake exits with the following error on MacOS:`crypto/include/internal/cryptlib.h:13:11: fatal error: 'stdlib.h' file not found`, run `export  MACOSX_DEPLOYMENT_TARGET=10.14`

2. Can we bring back the page that talks about Windows installation? I believe there were some steps to set up the environment in windows to run producer sdk samples. Quite a few customers are asking about the previous steps that were followed in Windows.